### PR TITLE
Publish blog post for Tenzir v4.7

### PR DESCRIPTION
**Pull request by dominiklohmann**
The usual. It's fluffy.

### CODE REVIEW

File: web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg
Comment (dominiklohmann) [inline web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: @mavam this still needs a new image—edit token is yours.

**Review by mavam — APPROVED** [review]

File: web/blog/tenzir-v4.7/index.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+```json {0} title="Example output"
+{
+  "message": {
+    "client": "55.3.244.1",
+    "method": "GET",
+    "request": "/index.html",
+    "bytes": 15824,
+    "duration": "0.043"
```
Comment (eliaskosunen) [inline web/blog/tenzir-v4.7/index.md:105]:
```suggestion
    "duration": 0.043
```

Comment (dominiklohmann) [reply web/blog/tenzir-v4.7/index.md:105]: Nice catch, thank you! I also updated the image accordingly.

File: web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg
Comment (eliaskosunen) [inline web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: At least my system can't render this SVG:

image
image

Comment (dominiklohmann) [reply web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: Do you see this when building the docs site locally or when viewing the svg on GitHub directly?

Comment (eliaskosunen) [reply web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: The svg itself, both on GitHub and locally after downloading it. Didn't try building the docs

Comment (dominiklohmann) [reply web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: See d2ba52b0d4

Comment (eliaskosunen) [reply web/blog/tenzir-v4.7/tenzir-v4.7.excalidraw.svg:1]: Seems to be working now, both on GitHub and locally :+1:

**Pull request merged**